fmlloader/src/main/java/net/minecraftforge/fml/loading/StringSubstitutor.java

```diff
@@ -10,37 +10,30 @@
 import org.apache.commons.lang3.text.StrLookup;
 import org.apache.commons.lang3.text.StrSubstitutor;
 
-import java.util.Map;
-
 @SuppressWarnings("deprecation")
-public class StringSubstitutor
-{
-    private static final Map<String,String> globals = ImmutableMap.of(
-            "mcVersion", FMLLoader.versionInfo().mcVersion(),
-            "forgeVersion", FMLLoader.versionInfo().forgeVersion()
-    );
-
+public class StringSubstitutor {
     public static String replace(final String in, final ModFile file) {
         return new StrSubstitutor(getStringLookup(file)).replace(in);
     }
 
     private static StrLookup<String> getStringLookup(final ModFile file) {
-        return new StrLookup<String>()
-        {
+        var globals = ImmutableMap.of(
+            "mcVersion", FMLLoader.versionInfo().mcVersion(),
+            "forgeVersion", FMLLoader.versionInfo().forgeVersion()
+        );
```

Comment:
That is correct, this has to be done because i can no longer pull these in in a static that could be called any time.
Ya.. could cache it, but does it really matter?

Reply:
In larger packs we could have over 300 mods, so possibly.

Also, I just realised this module's Java 17, so might as well use `java.util.Map#of` instead of guava's to benefit from the `@Stable` and `@ValueBased` internal optimisations in the JVM.